sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationTests.cs

```diff
@@ -492,5 +493,43 @@ public async Task AnalyzeOperationBatchWithStatisticsTest()
             Assert.AreEqual(51, result[0].Statistics.CharacterCount);
             Assert.AreEqual(1, result[0].Statistics.TransactionCount);
         }
+
+        [RecordedTest]
+        public async Task AnalyzeOperationAllActionsAndDisableServiceLogs ()
+        {
+            TextAnalyticsClient client = GetClient();
+
+            TextAnalyticsActions batchActions = new TextAnalyticsActions()
+            {
+                ExtractKeyPhrasesOptions = new List<ExtractKeyPhrasesOptions>() { new ExtractKeyPhrasesOptions() { DisableServiceLogs = true } },
+                RecognizeEntitiesOptions = new List<RecognizeEntitiesOptions>() { new RecognizeEntitiesOptions() { DisableServiceLogs = true } },
+                RecognizePiiEntitiesOptions = new List<RecognizePiiEntitiesOptions>() { new RecognizePiiEntitiesOptions() { DisableServiceLogs = false } },
+                RecognizeLinkedEntitiesOptions = new List<RecognizeLinkedEntitiesOptions>() { new RecognizeLinkedEntitiesOptions() { DisableServiceLogs = true } },
```

Comment:
This option only affects the behavior on the service side, but no changes can be observed from the client, correct?

Reply:
correct